gazebo8/releases/jade.yaml

```diff
@@ -4,5 +4,8 @@ gazebo_plugins:
   ubuntu: [ros-jade-gazebo8-plugins]
 gazebo_ros:
   ubuntu: [ros-jade-gazebo8-ros]
+gazebo_ros_control:
+  ubuntu: [ros-jade-gazebo8-ros-control]
+  debian: [ros-jade-gazebo8-ros-control]
```

Comment:
I think the `debian` line doesn't belong

Reply:
You're probably right.  I'll remove that one from Jade.